service/create/service.go

```diff
@@ -163,14 +174,33 @@ func (s *Service) Boot() {
 						return
 					}
 
+					// Create AWS client
+					s.awsConfig.Region = cluster.Spec.AWS.Region
+					awsSession, ec2Client := awsutil.NewClient(s.awsConfig)
+
+					// Create KMS key
+					kmsSvc := kms.New(awsSession)
+					key, err := kmsSvc.CreateKey(&kms.CreateKeyInput{})
```

Comment:
It's a key to encrypt the TLS keys which we fetch in the instances. It's unique for a cluster.